VERSION

```diff
@@ -1 +1 @@
-8.17.1
+9.0.0-prerelease
```

Comment:
We will have several of these as ES gets closer to the actual release candidates, so we should start with something that can be incremented and remain sortable.

```suggestion
9.0.0-prerelease01
```